app/code/Magento/ImportExport/Model/Import/ErrorProcessing/ProcessingErrorAggregatorInterface.php

```diff
@@ -79,6 +79,13 @@ public function getInvalidRowsCount();
      */
     public function initValidationStrategy($validationStrategy, $allowedErrorCount = 0);
 
+    /**
+     * Return the validation strategy
+     *
+     * @return string
+     */
+    public function getValidationStrategy();
```

Comment:
Unfortunately in this type of release (a PR to 2.2-develop is considered a Patch release) we cannot change interface classes because of our BiC guidelines. It would be a good idea to add a new interface and migrate any similar methods to this new interface (see http://devdocs.magento.com/guides/v2.2/contributor-guide/backward-compatible-development/index.html#introduction-of-a-method-to-a-class-or-interface for more information)